examples/allennlp/allennlp_jsonnet.py

```diff
@@ -4,19 +4,8 @@
 
 In this example, we optimize the validation accuracy of
 sentiment classification using an AllenNLP jsonnet config file.
-Since it is too time-consuming to use the training dataset,
-we here use the validation dataset instead.
```

Comment:
Just wondering but can we not keep this sentence?

Reply:
Ah I am sorry I thought in your previous review you referred to this sentence as well

Reply:
Ah, sorry if it wasn't clear. I only referred to the "exec" part, how to execute the examples.

Reply:
Ah no worries, thanks!